core/templates/domain/story/StoryContentsObjectFactorySpec.ts

```diff
@@ -44,7 +44,9 @@ describe('Story contents object factory', () => {
           destination_node_ids: ['node_2'],
           outline: 'Outline',
           exploration_id: null,
-          outline_is_finalized: false
+          outline_is_finalized: false,
+          thumbnail_bg_color: 'white',
```

Comment:
Can you make this hex and make sure it is the correct value for this domain object (see constants.ts)? (Do this in the topic, subtopic, and story node specs too.)

Reply:
done